pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -175,13 +193,14 @@ SEC("uprobe/SSL_write")
 int uprobe__SSL_write(struct pt_regs* ctx) {
     void *ssl_ctx = (void *)PT_REGS_PARM1(ctx);
     u64 pid_tgid = bpf_get_current_pid_tgid();
+    log_debug("uprobe/SSL_write: pid_tgid=%llx ctx=%llx\n", pid_tgid, ssl_ctx);
     conn_tuple_t *t = tup_from_ssl_ctx(ssl_ctx, pid_tgid);
     if (t == NULL) {
         return 0;
     }
 
     void *ssl_buffer = (void *)PT_REGS_PARM2(ctx);
-    size_t len = (size_t)PT_REGS_PARM3(ctx);
+    int len = (int)PT_REGS_PARM3(ctx);
```

Comment:
why don't we check the `write` function finished successfully? (a.k.a len >= 0)

Reply:
This `len` is an argument, we will need a return probe to check the return value.

Reply:
Added return probes and value checks